user-manager/src/test/java/com/qpay/usermanager/service/impl/CustomerServiceImplTest.java

```diff
@@ -37,6 +40,12 @@ class CustomerServiceImplTest {
     @Mock
     private CustomerMapper customerMapper;
 
+    @Mock
+    private UserMapper userMapper;
+
+    @Mock
+    private AuthenticationClient authenticationClient;
```

Comment:
you have to use it. Use `then` to validate you use it with correct data. Same for merchant